obs_status.yml

```diff
@@ -0,0 +1,2147 @@
+bad:
+  5115672: 9
+  33690776: 9
+  34996472: 9
+  38535536: 9
+  40113272: 9
+  45616472: 9
+  47846640: 9
+  48892720: 9
+  49818760: 9
+  62922104: 9
+  68956456: 9
+  92812824: 9
+  107881800: 9
+  109061768: 9
+  110493968: 9
+  111028544: 9
+  111807640: 9
+  118359864: 9
+  118498608: 9
+  120457376: 9
+  144714600: 9
+  145113736: 9
+  152831696: 9
+  161357600: 9
+  166993856: 9
+  185871616: 9
+  187829568: 9
+  188482256: 9
+  188501784: 9
+  188623712: 9
+  188750920: 9
+  195167088: 9
+  225576592: 9
+  239080480: 9
+  257953336: 9
+  264115840: 9
+  288753656: 9
+  306200176: 9
+  311824376: 9
+  327945264: 9
+  328077952: 9
+  330967424: 9
+  350392600: 11
+  373564272: 9
+  377102496: 9
+  391120072: 9
+  396501520: 9
+  405276920: 9
+  417611952: 11
+  444743456: 11
+  450103168: 9
+  458500128: 13  # new value
+  497824592: 9
+  505545160: 9
+  612370792: 9
+  614482704: 9
+  615257632: 9
+  615914552: 9
+  616699752: 9
+  626667256: 12
+  639635448: 9
+  643702120: 9
+  647762704: 9
+  649070072: 9
+  652480464: 9
+  653529952: 9
+  653919168: 9
+  654050016: 9
+  656409216: 9
+  664279632: 9
+  686557584: 9
+  686557784: 9
+  689441328: 9
+  690625640: 9
+  694561568: 9
+  720504096: 9
+  723657296: 12
+  740693864: 9
+  771756656: 9
+  805966472: 9
+  821318496: 9
+  836372744: 9
+  836381648: 9
+  879768704: 9
+  891159184: 9
+  897878136: 9
+  898125104: 9
+  899950912: 9
+  901664112: 9
+  904010616: 9
+  913441888: 9
+  915939232: 9
+  962998672: 9
+  974791856: 9
+  1058938112: 9
+  1092619768: 9
+  1105592952: 9
+  1108353064: 9
+  1117787424: 9
+  1130649544: 9
+  1139171624: 9
+  1178221128: 9
+  1197635184: 9
+mags:
+- agasc_id: 5115672
+  mag_aca: 11.422
+  mag_aca_err: 0.1
+- agasc_id: 33690776
+  mag_aca: 9.984
+  mag_aca_err: 0.1
+- agasc_id: 34996472
+  mag_aca: 10.562
+  mag_aca_err: 0.1
+- agasc_id: 38535536
+  mag_aca: 10.719
+  mag_aca_err: 0.1
+- agasc_id: 40113272
+  mag_aca: 11.781
+  mag_aca_err: 0.1
+- agasc_id: 45616472
+  mag_aca: 11.422
+  mag_aca_err: 0.1
+- agasc_id: 47846640
+  mag_aca: 10.891
+  mag_aca_err: 0.1
+- agasc_id: 48892720
+  mag_aca: 11.484
+  mag_aca_err: 0.1
+- agasc_id: 49818760
+  mag_aca: 10.859
+  mag_aca_err: 0.1
+- agasc_id: 62922104
+  mag_aca: 10.891
+  mag_aca_err: 0.1
+- agasc_id: 68956456
+  mag_aca: 11.328
+  mag_aca_err: 0.1
+- agasc_id: 82972168
+  mag_aca: 11.2
+  mag_aca_err: 0.1
+- agasc_id: 92812824
+  mag_aca: 11.141
+  mag_aca_err: 0.1
+- agasc_id: 107881800
+  mag_aca: 10.969
+  mag_aca_err: 0.1
+- agasc_id: 109061768
+  mag_aca: 11.621  # need to check, not from starcheck report
```

Comment:
I think the starcheck report at

https://icxc.harvard.edu/mp/mplogs/2019/FEB0419/oflsa/starcheck.html#obsid48456

is probably right, which looks like 11.797.